docs/pages/guides/tutorials/deploying-php-app-with-redis.md

```diff
@@ -0,0 +1,108 @@
+
+> **NOTE:** If this is your first time working with Meshery Playground, consider starting with the [Exploring Kubernetes Pods with Meshery Playground](https://docs.meshery.io/guides/tutorials/kubernetes-pods) tutorial first.
+
+### Prerequisites
+- Basic understanding of Kubernetes concepts.
+- Meshery Playground access. If you don't have an account, sign up at [Meshery Playground](https://meshery.layer5.io/play).
+
+### Lab Scenario
+- Import the PHP and Redis manifest files into Meshery Playground.
+- Deploy these resources on the playground.
+    - Start up a Redis leader.
+    - Start up two Redis followers.
+    - Start up the guestbook frontend.
+- Expose and view the Frontend Service.
+
+
+### Objective
+Learn how to import manifest files, visualize Kubernetes resources, create new resource components, and deploy the application using Meshery Playground.
+
+
+### Steps
+
+#### Download the Kubernetes Configuration Files
+
+To get started we will need some yaml files that will contain the configurations for the pods and services that will run on Meshery. You can download them to see what the look like.
+1. [redis-leader-deployment.yaml](https://k8s.io/examples/application/guestbook/redis-leader-deployment.yaml)
+2. [redis-leader-service.yaml](https://k8s.io/examples/application/guestbook/redis-leader-service.yaml)
+3. [redis-follower-deployment.yaml](https://k8s.io/examples/application/guestbook/redis-follower-deployment.yaml)
+4. [redis-follower-service.yaml](https://k8s.io/examples/application/guestbook/redis-follower-service.yaml)
+5. [frontend-deployment.yaml](https://k8s.io/examples/application/guestbook/frontend-deployment.yaml)
+6. [frontend-service.yaml](https://k8s.io/examples/application/guestbook/frontend-service.yaml)
+
+These YAML files contain the Service definitions and Deployment configurations for the PHP app with Redis.
+
+#### Accessing Meshery Playground
+
+1. Log in to the [Meshery Playground](https://meshery.layer5.io/) using your credentials. On successful login, you should be at the dashboard. Press the **X** on the _Where do you want to start?_ popup to close it (if required).
+
+2. Click **Explore** in the Cloud Native Playground tile to navigate to _MeshMap_
+
+
+#### Import the Files to Meshery Playground
+
+1. In the left sidebar, click on the upward arrow symbol(import icon) to import the designs into Meshery.
+
+2. In the modal that appears:
+   - Enter a name for your design in the "Design File Name" field (e.g.`redis-leader-deployment`).
+
+   - Select `Kubernetes Manifest` from the "Design Type" dropdown menu.
+
+   - Choose `URL Import` for the upload method, and input the URL.
+
+   - Then, click on `Import`
+
+        ![Import redis-leader-deployment](./screenshots/redis-leader-deployment.png)
```

Comment:
The screenshot uses number 2 to 6 but in the tutorial they are simple bullets. This can cause confusion. If we are numbering steps in screenshots, the written steps should preferably match them.
![2024-09-24_19-36](https://github.com/user-attachments/assets/bc88b2e6-3bb7-4add-882b-f85e7c855d91)